web/pdf_page_view.js

```diff
@@ -1001,6 +992,7 @@ class PDFPageView {
       if (isCanvasHidden) {
         canvas.hidden = false;
         isCanvasHidden = false;
+        canvasWrapper.classList.add("loading");
```

Comment:
Why is this necessary now, since I don't think it should be?
(If this is actually needed, that feels like a bug elsewhere...)

Reply:
I added an explanation here:
https://github.com/mozilla/pdf.js/pull/15992/files#diff-c48e3561004f5db8f11d5ebab2fd661591222ba911cb4173fbced15f026bac6bR1001

I think it's better to have it in the setTimeout in order to have an almost respected delay.